docs/architecture/cloud-native/infrastructure-as-code.md

```diff
@@ -1,20 +1,22 @@
 ---
 title: Infrastructure as code
 description: Architecting Cloud Native .NET Apps for Azure | Infrastructure As Code
-ms.date: 06/30/2019
+ms.date: 05/03/2020
 ---
 
 # Infrastructure as code
 
 [!INCLUDE [book-preview](../../../includes/book-preview.md)]
 
-Cloud-native applications tend to make use of all sorts of fantastic platform as a service (PaaS) components. On a cloud platform like Azure, these components might include things like storage, Service Bus, and the SignalR service. As applications become more complicated, the number of these services in use is likely to grow. Just as how continuous delivery broke the traditional model of deploying to an environment manually, the rapid pace of change also broke the model of having a centralized IT group manage environments.
+Cloud-native applications favor platform as a service (PaaS) resources. In the Azure cloud, these services include things like storage, Service Bus, and the SignalR service. As applications become more complex, the number of services they consume will grow. Just as continuous delivery automated the traditional model of manual deployments, Infrastructure as Code (IaC) is evolving how application environments are managed.
```

Comment:
I think, we should re-write this intro to IaC. While reading I was not able to relate why IaC is relevant because of Cloud-native apps favoring PaaS resources?

Reply:
Fixed

Reply:
Fixed